README.md

```diff
@@ -11,23 +11,22 @@
 
 <!-- usage note: edit this section. -->
 
-This is the manuscript for a community-written [Ten Simple
-Rules](https://collections.plos.org/ten-simple-rules) article aimed at [PLOS
+This is the manuscript for a community-written article aimed at [PLOS
 Computational Biology](https://journals.plos.org/ploscompbiol/). Deep learning is exploding in
 popularity and is increasingly finding its way into biological data analysis.
 However, since most biologists receive little (or no) data science training,
 using deep learning properly can be a daunting task.
 
 **We're going to try to fix that.** By boiling down the community's knowledge
-into ten simple rules, we hope to increase the number of biological researchers
+into ten quick tips, we hope to increase the number of biological researchers
 using DL (by making it more inviting) and the quality of the research (by
 helping them avoid common mistakes).
 
 Join us!
 
 ## Where we are now
 
-Currently, we're in the process of drafting the paper with each rule being tracked in its own [project](https://github.com/Benjamin-Lee/deep-rules/projects). To contribute to the paper, take a look at [`CONTRIBUTING.md`](CONTRIBUTING.md) for information on how we're organizing the writing as well as [`USAGE.md`](USAGE.md) for instructions on how to write the actual text.
+Currently, we're in the process of drafting the paper with each tip being tracked in its own [project](https://github.com/Benjamin-Lee/deep-rules/projects). To contribute to the paper, take a look at [`CONTRIBUTING.md`](CONTRIBUTING.md) for information on how we're organizing the writing as well as [`USAGE.md`](USAGE.md) for instructions on how to write the actual text.
```

Comment:
This would also be a good time to note in the readme that the pre-submission inquiry was approved with a link to the issue.